db/seeds.rb

```diff
@@ -43,6 +43,27 @@
                                school_type: "Academy",
                                gias_data: { 'ReligiousCharacter (name)': "None" })
 
+# A second school at the local authority to enable seeding a vacancy at multiple schools.
+school_three = FactoryBot.create(:school,
+                                 name: "Townhill Junior School",
+                                 urn: "116134",
+                                 phase: :primary,
+                                 url: "http://www.townhilljuniorschool.co.uk",
+                                 minimum_age: 7,
+                                 maximum_age: 11,
+                                 address: "Benhams Road",
+                                 town: "Southampton",
+                                 county: "Hampshire",
+                                 postcode: "SO18 2NX",
+                                 region: "London",
+                                 easting: "445283",
+                                 northing: "114779",
+                                 geolocation: "(50.9306936449461,-1.3569968052135)",
+                                 readable_phases: %w[primary],
+                                 detailed_school_type: "Foundation school",
+                                 school_type: "Local authority maintained schools",
+                                 gias_data: { 'ReligiousCharacter (name)': "Does not apply" })
```

Comment:
What's the reason for single quotes here?

Reply:
I blame Cesi